### ISSUE TITLE
Clarify Lint/UnneededSplatExpansion's message

### ISSUE THREAD

**Issue by mikegee** [rubocop-hq/rubocop#6151]
The message from `Lint/UnneededSplatExpansion` doesn't make it clear how to resolve the problem. It seems like the message is telling the developer to only remove the splat and leave the array. But, that changes the behavior. Running autocorrect is required to understand what Rubocop is asking for.

--------

## Expected behavior

Code like this
```ruby
case status
when *%i[halt stop]
  make_it_stop
end
```
should give a clear message about the solution. The output should be something more like:
```
$ rubocop splat_rubocop.rb
Inspecting 1 file
W

Offenses:

splat_rubocop.rb:9:6: W: Lint/UnneededSplatExpansion: Replace splat expansion with comma separated values.
when *%i[halt stop]
     ^^^^^^^^^^^^^^

1 file inspected, 1 offense detected
```

## Actual behavior

Code like this
```ruby
case status
when *%i[halt stop]
  make_it_stop
end
```
gives an unclear message without any clues on how to resolve it:
```
$ rubocop splat_rubocop.rb
Inspecting 1 file
W

Offenses:

splat_rubocop.rb:9:6: W: Lint/UnneededSplatExpansion: Unnecessary splat expansion.
when *%i[halt stop]
     ^^^^^^^^^^^^^^

1 file inspected, 1 offense detected
```
Running autocorrect is required to see what Rubocop wants:
```ruby
case status
when :halt, :stop
  make_it_stop
end
```

## Steps to reproduce the problem

Run Rubocop on this file:
```ruby
case status
when *%i[halt stop]
  make_it_stop
end
```

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.58.2 (using Parser 2.5.1.0, running on ruby 2.5.1 x86_64-darwin16)
```

**Comment by jaredbeck** [comment]
> The output should be something more like:
> W: Lint/UnneededSplatExpansion: Replace splat expansion with comma separated values.

How about:

> W: Lint/UnneededSplatExpansion: The array is unnecessary because its elements are literal values.

Or say both.

The former is "give a person a fish", the latter is (hopefully) "teach a person to fish".

**Comment by rrosenblum** [comment]
This isn't the first time that someone has brought up messaging of this cop being unclear. The proposed verbiage changes make sense to me.

**Comment by Drenmi** [comment]
On a side note, is this actually a `Lint` cop, or should it be demoted to `Style`?
